typings/xterm-player.d.ts

```diff
@@ -12,6 +58,11 @@ declare module 'xterm-player' {
      */
     url: string
 
+    /**
+     * The options of the player
+     */
+    playerOptions: IPlayerOptions
+
```

Comment:
Currently I don't want to expose playerOptions at here. 
Because this value is not responsive, hence if you change any value at here it won't effect anything.

Reply:
Notice the fact that, some options of xterm.js can only be specified at the very beginning, I will change playerOptions to be one-time configuration.